facade/src/main/resources/buildConfigurationSupportedGenericParameters.properties

```diff
@@ -17,6 +17,6 @@
 #
 
 CUSTOM_PME_PARAMETERS=Additional parameters, which will be passed to the PME CLI executable during the alignment before the build. The format is as you would enter them on a command line, and each MUST start with a dash.
-EXECUTION_ROOT_NAME=Specify the execution root name of the build configuration. This is used to override the default value, and can be useful for builds that disable PME. Format is '<groupid>:<artifactid>'
+BREW_BUILD_NAME=Specify the execution root name of the build configuration. This is used to override the default value, and can be useful for builds that disable PME. Format is '<groupid>:<artifactid>'
```

Comment:
Mabye also change the description?
And the format '<groupid>:<artifactid>' is only for maven builds. There is no specified format for other type of builds.